wiki/Client/Menu_tips/fr.md

```diff
@@ -0,0 +1,121 @@
+- Des spinners trop longs fatiguent les joueurs ; trop courts, ils les prennent par surprise. Limitez leur durée à une poignée de secondes seulement et évitez d'en placer beaucoup successivement.
+- Faites un `clic droit` sur un objet pour le supprimer. Commencez un nouveau [combo](/wiki/Beatmapping/Combo) en faisant un `clic droit` dans une zone vide.
+- Pensez à placer des bookmarks aux points critiques de votre beatmap, tels que le début d'un refrain, d'un couplet, d'un interlude, d'une pause ou d'une section où le timing change.
+- Utilisez les touches `J` et `K` de votre clavier pour déplacer les objets sélectionnés d'un cran en arrière ou en avant dans la timeline.
+- Appuyez sur `F5` ou cliquez sur le bouton `Test` dans le bas de l'écran pour tester votre beatmap à partir de la section dans laquelle vous êtes. [Tester vos beatmaps](/wiki/Client/Beatmap_editor/Test_mode) vous permet d'identifier des notes mal placées et une éventuelle baisse trop conséquente de la barre de vie.
+- Zoomez dans la timeline en cliquant sur les boutons `+` / `-` en haut à gauche de l'écran, ou en utilisant la combinaison `Alt` + `molette` de la souris tout en survolant la timeline avec le curseur. Ceci vous aidera à la lire plus facilement dans le cas où elle serait surchargée.
+- N'hésitez pas à ralentir la vitesse pour revérifier le timing et le positionnement des notes. Utilisez les boutons `25%` - `100%` en bas à droite de l'écran, les touches `Haut` et `Bas` de votre clavier ou les combinaisons `Ctrl` + `Haut` et `Ctrl` + `Bas`.
+- Vous avez décidé d'abandonner une beatmap ? Pas de problème, toutes les beatmaps restées inactives pendant un certain temps sont automatiquement envoyées dans le [cimetière](/wiki/Beatmap/Category#cimetière) (`graveyard`). Si vous souhaitez ressusciter une map, cliquez sur `Resurrect` à côté de son nom dans votre profil web.
+- Une beatmap peut posséder son propre skin intégré, il suffit juste de copier les éléments du skin dans le dossier de la beatmap. Gardez la même convention de nommage que pour les skins ordinaires.
+- Saviez-vous qu'en copiant des notes dans l'éditeur (`Ctrl` + `C` ou menu `Edit`), le presse-papier contiendra leur position temporelle ainsi que le numéro du combo ? C'est très utile pour rédiger des posts dans le forum et faire des mods !
+- Utilisez le live mapping (`Ctrl` + `Tab`) pour placer très rapidement des notes avec les touches que vous utilisez habituellement pour jouer à osu!mania et Taiko.
+- Faites attention à ne pas superposer un slider avec un autre ou avec lui-même pour satisfaire aux critères de ranking.
+
+## osu!(lazer)
+
+<!-- Source: https://github.com/ppy/osu/blob/master/osu.Game/Screens/Menu/Disclaimer.cs -->
+
+Ces astuces apparaissent dans la [version lazer du client](https://github.com/ppy/osu-lazer), au démarrage, avant la séquence d'introduction.
```

Comment:
as mentioned in the top comment, add a sentence here that says these tips aren't translated in-game

Reply:
> as mentioned in the top comment, add a sentence here that says these tips aren't translated in-game

Done, i'll just ask a review of ganondorf